.gitattributes

```diff
@@ -1,10 +1,12 @@
-.editorconfig     export-ignore
-.gitattributes    export-ignore
-/.github/         export-ignore
-.gitignore        export-ignore
-/.travis.yml      export-ignore
-/build/           export-ignore
-/docs/            export-ignore
-/Makefile         export-ignore
-/phpunit.xml.dist export-ignore
-/tests/           export-ignore
+/.editorconfig         export-ignore
```

Comment:
Some files, like editorconfig and gitignore were ignored from the entire tree, not just the root on purpose

Reply:
None of the files exist in any subdirectory though